Intelipro/reja
# Fix: Remove Code Splitting Loader everywhere

#### What does this PR do?
 Fixes code-splitting everywhere

#### Description of Task to be completed?
- [x]  Removes the `React.lazy` function in each component in the views
- [x] Add the `React.lazy` function on each view as a whole 
- [x] Remove `DialogContent` component from the preview modal

#### How should this be manually tested?
- Clone this repo `git clone  [the-repo-link]`
- Checkout to this branch `bg-fix-code-splitting`
- Install all dependencies `npm install`
- Start the server `npm run build`
-  You'll notice  multiple bundles have been created that can be dynamically loaded at runtime

#### Any background context you want to provide?
- The fallback loader was being called each time a component is rendered
- The `DialogContent` made the modal to cover the whole page

#### What are the relevant pivotal tracker stories?

#### Screenshots (if appropriate)

#### Questions:
